src/js/tools/drawing/Rectangle.js

```diff
@@ -20,10 +20,19 @@
    * @override
    */
   ns.Rectangle.prototype.draw = function (col, row, color, targetFrame, penSize) {
-    var rectanglePixels = pskl.PixelUtils.getBoundRectanglePixels(this.startCol, this.startRow, col, row);
+    var rectangle = pskl.PixelUtils.getOrderedRectangleCoordinates(this.startCol, this.startRow, col, row);
```

Comment:
Can you just remove getBoundRectanglePixels from PixelUtils since no other file is using it now?